client/payment-gateways/style.scss

```
@@ -0,0 +1,34 @@
+.disable-confirmation-modal {
+	&__payment-methods-list {
+		margin-bottom: 32px;
+
+		> * {
+			&:not( :last-child ) {
+				margin-bottom: 12px;
+			}
+		}
+	}
+}
+
+.wcstripe-confirmation-modal {
+
+	&.alert {
```

Just so that the spacing between blocks is applied consistently

```suggestion
.wcstripe-confirmation-modal {
	&.alert {
```

---

Ah, yep. Missed that - thank you.